experimental/pedometer/README.md

```diff
@@ -38,7 +38,7 @@ Note that step counting is only available on physical devices.
 
 ## Project stucture
 
-* `src`: Contains the native source code, and a CmakeFile.txt file for building
+* `src`: Contains the native source code, and a CMakeFile.txt file for building
```

Comment:
I suspect this is meant to be `CMakeLists.txt`

Reply:
Made the correction to CMakeLists.txt